spdmlib/src/common/mod.rs

```diff
@@ -524,6 +525,7 @@ pub struct SpdmRuntimeInfo {
     pub need_measurement_signature: bool,
     pub message_a: ManagedBuffer,
     pub message_m: Option<HashCtx>,              // for M1/M2
+    pub message_l: Option<HashCtx>, // for measurement signature L1/L2. DSP0274: MEASUREMENTS signature generation
```

Comment:
please rename message_m to digest_context_m1m2.

please rename message_l to digest_context_l1l2.